### DISCUSSION TITLE
Should the MongoDB CXX driver source be removed from the docker image?

### DISCUSSION THREAD

**Discussion by ivanperez-keera** [space-ros/docker#105] [Ideas]
The docker image installs `mongo-cxx-driver-r3.6.7`.

https://github.com/space-ros/docker/blob/76cd412a7379dd5a5ec338b7b75659afbbf0c9a1/space_robots/Dockerfile#L72-L73

Installation is placed in `/usr/local`. Therefore, the `tar.gz` and the directory where the source was unpacked are no longer needed.

It would make for a cleaner Space ROS image to have those removed. I was just now connecting to a Space ROS image via VSCode's file explorer and having that in the folder was unnecessary noise.

I don't want to pollute our issues with too many ideas that we will never do much about, so I'm filing this first here to see what people think. If we like this and plan to act on it, we can promote it to issue.

**Comment by Bckempa** [comment]
I discussed this with @EzraBrooks last month, the MongoDB components are the vast majority of the build time for the `space-robots` image right now but are only needed by [`warehouse`](https://github.com/ros-planning/warehouse_ros). We should verify if this even needs to be in the image first.
